src/adams_methods.jl

```diff
@@ -0,0 +1,626 @@
+using ProgressMeter
+"""
+    t,y = ode_ms(odefun, y0, tspan, order=4) with tspan = [t0:dt:tfinal]
+
+Fixed-Equal Steps Adams-Bashforth multi-step solver for nonstiff problems
+
+(Main ref: Hairer & Wanner 1996, Vol I, p.359-360, 372)
+
+Note: If method is run at order k, the first step is taken at order 1,
+and the order increases each step until it is order k on the kth step.
+Thus results may be of lower accuracy than expected.
+"""
+
+function ode_ms(F, y0, tspan, order::Integer)
```

Comment:
These should have a trailing `;kwargs...` to match the fixes of https://github.com/JuliaDiffEq/ODE.jl/pull/119